sdk/tests/js/webgl-test-utils.js

```diff
@@ -2073,6 +2086,24 @@ var loadStandardProgram = function(gl) {
   return program;
 };
 
+var loadStandardProgramAsync = function(gl, callback) {
+  loadStandardVertexShaderAsync(gl, function(gl) {
+    return function(success, vs) {
+      loadStandardFragmentShaderAsync(gl, function(vs) {
+        return function(success, fs) {
+          var program = gl.createProgram();
+          gl.attachShader(program, vs);
+          gl.attachShader(program, fs);
+          gl.bindAttribLocation(program, 0, "a_vertex");
+          gl.bindAttribLocation(program, 1, "a_normal");
+          linkProgram(gl, program);
+          callback(true, program);
```

Comment:
Same basic problem here. First, "success" is being ignored multiple times. Second, "callback" is never called with "false" as the first argument, so callers can not be robust to errors.